docs/usage_general.rst.inc

```diff
@@ -202,6 +202,7 @@ General:
         use fqdn@uniqueid.
     BORG_LOGGING_CONF
         When set, use the given filename as INI_-style logging configuration.
+        A basic example conf can be found at ``docs/misc/logging.conf``.
```

Comment:
For a basic example configuration, see: ...

(wasn't sure about "at" or "in", so avoided it)